.github/workflows/pr.yml

```diff
@@ -0,0 +1,36 @@
+# Publish a -beta version to npm
+
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        id: checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        id: node
+        uses: actions/setup-node@v2
+        env:
+          PLAYER_NPM_PUBLISH: ${{ secrets.PLAYER_NPM_PUBLISH }}
+        with:
+          node-version: '16'
+          cache: 'npm'
+
+      - name: Publish npm package
+        env:
+          PLAYER_NPM_PUBLISH: ${{ secrets.PLAYER_NPM_PUBLISH }}
+        working-directory: package
+        run: |
+          npm --version
+          echo '_authToken=${PLAYER_NPM_PUBLISH}' > .npmrc
+          npm run publish --tag beta
```

Comment:
Before you publish, you'll need to do something like this:

```
npm run release -- --no-verify --prerelease beta.${RUN_NUMBER}.${SHA:0:8}
```

Otherwise it'll try to publish over the version we're making changes to and fail. This will produce a unique version number each time the PR updates, based on the PR's run ID.

Reply:
Thanks @maslade yeah I thought --tag would also add -beta to the version, but even if it did multiple PR commits would fail.

Now I explicitly set the version to <current version>.beta.<run number>.<run attempt> so that a PR can have multiple commits and jobs and be re-ran without failure